beep/protocol/procedure_templates/a_formation06162921_Xiao.000

```diff
@@ -0,0 +1,611 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
@willpowelson Could you remove this file from the PR? I think it might be better to keep it out of the repo since it involves ongoing work and might be confusing... :)

Reply:
@patrickherring-TRI  it's gone!